test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js

```diff
@@ -0,0 +1,126 @@
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/poll')
+  .respond(xhrSuccess);
+
+fixture('Enroll Okta Verify Authenticator');
+
+async function setup(t) {
+  const enrollOktaVerifyPage = new EnrollOktaVerifyPageObject(t);
+  await enrollOktaVerifyPage.navigateToPage();
+
+  const { log } = await t.getBrowserConsoleMessages();
+  await t.expect(log.length).eql(3);
+  await t.expect(log[0]).eql('===== playground widget ready event received =====');
+  await t.expect(log[1]).eql('===== playground widget afterRender event received =====');
+  await t.expect(JSON.parse(log[2])).eql({
+    controller: null,
+    formName: 'enroll-poll',
+    authenticatorType: 'app',
+  });
+
+  return enrollOktaVerifyPage;
+}
+
+test.requestHooks(logger, enrollViaQRcodeMocks)('should be able to enroll via qrcode', async t => {
+  const enrollOktaVerifyPage = await setup(t);
+  await t.expect(enrollOktaVerifyPage.getFormTitle()).eql('Set up Okta Verify');
+  await t.expect(enrollOktaVerifyPage.hasEnrollViaQRInstruction()).eql(true);
+  await t.expect(enrollOktaVerifyPage.hasQRcode()).eql(true);
+  await t.expect(enrollOktaVerifyPage.hasSwitchChannelLink()).eql(true);
+  await t.wait(4000);
+  await t.expect(logger.count(
+    record => record.response.statusCode === 200 &&
+    record.request.url.match(/poll/)
+  )).eql(1);
+  const successPage = new SuccessPageObject(t);
+  const pageUrl = await successPage.getPageUrl();
+  await t.expect(pageUrl)
+    .eql('http://localhost:3000/app/UserHome?stateToken=mockedStateToken123');
+});
+
+test.requestHooks(mock)('should render switch channel view when Can\'t scan is clicked', async t => {
+  const enrollOktaVerifyPage = await setup(t);
+  await enrollOktaVerifyPage.clickSwitchChannel();
+  const switchChannelPageObject = new SwitchOVEnrollChannelPageObject(t);
+  await t.expect(switchChannelPageObject.getFormTitle()).eql('More options');
+  await t.expect(switchChannelPageObject.getOptionCount()).eql(2);
+  await t.expect(switchChannelPageObject.getOptionLabel(0))
+    .eql('Text me a setup link');
+  await t.expect(switchChannelPageObject.getOptionLabel(1))
+    .eql('Email me a setup link');
+});
+
+test.requestHooks(enrollViaEmailMocks)('should be able enroll via email', async t => {
+  const enrollOktaVerifyPage = await setup(t);
+  await enrollOktaVerifyPage.clickSwitchChannel();
+  const switchChannelPageObject = new SwitchOVEnrollChannelPageObject(t);
+  await t.expect(switchChannelPageObject.getFormTitle()).eql('More options');
+  await switchChannelPageObject.selectChannelOption(1);
+  await switchChannelPageObject.clickNextButton();
+  const enrollViaEmailPageObject = new EnrollOVViaEmailPageObject(t);
+  await t.expect(enrollViaEmailPageObject.getFormTitle()).eql('Set up Okta Verify via email link');
+  await enrollViaEmailPageObject.fillEmailField('test@gmail.com');
+  await enrollViaEmailPageObject.clickNextButton();
+  const successPage = new SuccessPageObject(t);
```

Comment:
shall you verify the intermittent page (`check your email, 1.. 2.. 3..`) before success?